planning/behavior_path_planner/src/scene_module/avoidance/avoidance_module.cpp

```diff
@@ -2195,6 +2208,8 @@ void AvoidanceModule::addShiftPointIfApproved(const AvoidPointArray & shift_poin
       turn_signal_info.turn_signal.command = TurnIndicatorsCommand::ENABLE_RIGHT;
     }
 
+    ego_velocity_starting_avoidance_ptr_.reset();
```

Comment:
Is it okay if the behavior_path_planner is already in the flow of avoidance and then  another avoidance flow is approved

Reply:
seems this code should be put in a place where we check if deceleration is needed?

Reply:
Maybe, is it enough to remove this line?

Reply:
I removed this line when I resolve conflict...
https://github.com/autowarefoundation/autoware.universe/pull/909/commits/d3ef9e7ea7935e9e987c5d4b1fa3abca06dc6f14